helix-lsp/src/client.rs

```diff
@@ -243,6 +246,15 @@ impl Client {
         Ok((client, server_rx, initialize_notify))
     }
 
+    fn resolve_cmd_path(cmd: &str) -> Result<PathBuf> {
+        if Path::new(cmd).exists() {
```

Comment:
This check should be whether the path is absolute rather than whether the file exists. Otherwise files from the CWD will be prioritized over those from the path.